app/services/vacancy_facets.rb

```diff
@@ -0,0 +1,66 @@
+class VacancyFacets
+  FIELDS = %i[job_roles subjects cities counties].freeze
+
+  def initialize(store: Redis.new)
+    @store = store
+  end
+
+  def get(field)
+    return {} if !FIELDS.include?(field) || !store.exists?(field)
```

Comment:
```ruby
return {} unless FIELDS.include?(field) && store.exists?(field)
```
Might be more readable/ruby, but I don't mind either

Reply:
Yup that makes sense